Section 4 - Terraform Modules & Workspaces/module-outputs/projects/C/my-sg.tf

```diff
@@ -3,11 +3,40 @@ module "sgmodule" {
 }
 
 resource "aws_instance" "web" {
```

Comment:
**Tenable.cs remediation**
 
**[MEDIUM]: Ensure virtual private cloud (VPC) is configured for AWS EC2 instances**
 
**Vulnerability**
By default AWS EC2 instances are launched in default VPC which is not recommended for production grade. AWS VPCs provides the controls to facilitate a formal process for approving and testing all network connections and changes to the firewall and router configurations.
 
**Remediation**
Amazon Virtual Private Cloud (Amazon VPC) enables you to define a virtual network in your own logically isolated area within the AWS cloud, known as a virtual private cloud or VPC. You can create AWS resources, such as Amazon EC2 instances, into the subnets of your VPC. Your VPC closely resembles a traditional network that you might operate in your own data center, with the benefits of using scalable infrastructure from AWS.

source: tenablecs-